packages/compass-preferences-model/src/preferences-schema.ts

```diff
@@ -115,6 +115,15 @@ export type NonUserPreferences = {
   password?: string;
 };
 
+export type AtlasProjectPreferences = {
```

Comment:
this was a nit from a previous ticket that didn't get committed to the branch before it got merged so i just added it into this one (it was just shifting these two preferences up in the file for code cleanliness)